Sources/Core/AWSSDKIdentity/InternalClients/InternalAWSSSO/Dependencies.json

```diff
@@ -0,0 +1,15 @@
+[
+    "AWSClientRuntime",
+    "AWSSDKChecksums",
+    "AWSSDKHTTPAuth",
+    "ClientRuntime",
+    "Smithy",
+    "SmithyHTTPAPI",
+    "SmithyHTTPAuthAPI",
+    "SmithyIdentity",
+    "SmithyJSON",
+    "SmithyReadWrite",
+    "SmithyRetries",
+    "SmithyRetriesAPI",
+    "SmithyTestUtil"
+]
```

Comment:
Above is an example of the new JSON that is generated for each client, naming its dependencies.